repos/spack_repo/builtin/packages/py_alembic/package.py

```diff
@@ -14,13 +14,20 @@ class PyAlembic(PythonPackage):
 
     license("MIT")
 
+    version("1.16.5", sha256="a88bb7f6e513bd4301ecf4c7f2206fe93f9913f9b48dac3b78babde2d6fe765e")
     version("1.5.5", sha256="df0028c19275a2cff137e39617a39cdcdbd1173733b87b6bfa257b7c0860213b")
     version("1.0.7", sha256="16505782b229007ae905ef9e0ae6e880fddafa406f086ac7d442c1aaf712f8c2")
 
     depends_on("python@2.7:2.8,3.6:", type=("build", "run"))
```

Comment:
Missing dependency on `depends_on("python@3.9:", type=("build", "run"))` for version `@1.15:`

Reply:
same reaction as authlib, agreed should be added thx